packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -675,11 +675,18 @@ export class BackgroundScript {
       emittedPending = true
     }
 
+    const setUnavailable = (fromNo: 'token' | 'subscription') => {
```

Comment:
![image](https://user-images.githubusercontent.com/525211/122523417-3379bd00-d041-11eb-924d-91beae6b01bb.png)

Otherwise it gets stuck in a green circle outline "connecting" state